### PR TITLE
Add Odin icon

### LINKED ISSUE #12133: Request: Odin Programming Language
### Brand Name

Odin Programming Language

### Website

https://odin-lang.org/

### Popularity Metric

The Similarweb global rank is 447,974. See: https://www.similarweb.com/website/odin-lang.org/

### Official Resources for Icon and Color

Logo PNG: https://github.com/odin-lang/Odin/blob/master/misc/logo-slim.png

Color is probably dark blue - HSL(211, 63%, 32%) which is one of the https://odin-lang.org/ background colors

### Additional Comments

Language repository has BSD-3 license, no explicit license for logo, but I assume it's the same

### PR DESCRIPTION
**Pull request by Anonav0**
odinprogramminglanguage

**Issue:** closes #12133

**Popularity metric:** Github stars : 7k

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Odin is a general-purpose programming language with distinct typing, built for high performance, modern systems, and built-in data-oriented data types. The Odin Programming Language, the C alternative for the joy of programming.

### CODE REVIEW

File: _data/simple-icons.json
Comment (uncenter) [inline _data/simple-icons.json:1]: Please don't change any other formatting in this file.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json
Comment (Anonav0) [reply _data/simple-icons.json:1]: So will this get merged?

Comment (uncenter) [reply _data/simple-icons.json:1]: You need to revert the formatting diff please.

Comment (Anonav0) [reply _data/simple-icons.json:1]: Understood where's the problem at. Could you tell me how can solve this re formatting issue? Should I disable Perttier and try again? Sorry for bothering this much.

Comment (Anonav0) [reply _data/simple-icons.json:1]: I updated the file. and made a new commit to it. Should I close this PR and create a new one with all the necessary changes?

Comment (uncenter) [reply _data/simple-icons.json:1]: No need for a new PR, just push the commit with the changes to this branch.

Comment (Anonav0) [reply _data/simple-icons.json:1]: Done!

Comment (uncenter) [reply _data/simple-icons.json:1]: Great thank you!

**Review by uncenter — CHANGES_REQUESTED** [review]
Actually, sorry for the hassle - let's rename the icon to "Odin", which will mean moving the `icons/odinprogramminglanguage.svg` file to `icons/odin.svg`, and updating the `<title>` element inside the SVG.

**Comment by Anonav0** [comment]
Alright

**Comment by Anonav0** [comment]
Made the changes.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11017,6 +11017,11 @@
             "source": "https://en.facebookbrand.com/oculus/assets/oculus?audience=oculus-landing",
             "guidelines": "https://en.facebookbrand.com/oculus/"
         },
+        {
+            "title": "Odin",
+            "hex": "1E5085",
```
Comment (uncenter) [inline _data/simple-icons.json:11022]:
```suggestion
            "hex": "3882D2",
```
My apologies, I actually think the hex code should be this based on everything except the header of https://odin-lang.org/, and all the colors on https://pkg.odin-lang.org/.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Pull request merged**